scripts/variantstore/wdl/extract/summarize_task_monitor_logs.py

```diff
@@ -125,8 +125,12 @@ def parse_cpu_usage_line(line):
         p2 = "^\\* CPU usage\\: -nan\\%$"  # * CPU usage: -nan%
         m2 = re.match(p2, line)
         if m2 is None:
-            eprint(f"ERROR: Line '{line}' does not look like a CPU usage line. Is this a monitoring_log file?")
-            sys.exit(1)
+            # Check if it's just empty
+            p3 = "^\\* CPU usage\\:$"  # * CPU usage:
```

Comment:
This is in fact the pattern I saw in my error message, though given the [relevant echo statement](https://github.com/broadinstitute/dsp-scripts/blob/master/cromwell/methods/cromwell_monitoring_script.sh#L237) in the monitoring script I'm not sure why there isn't a space after the `:` 🤔  This might be more robust with a little slack added in if that space does decide to show up

```suggestion
            p3 = "^\\* CPU usage\\: *$"  # * CPU usage:
```